server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/Repository.java

```diff
@@ -94,10 +97,57 @@ default Author author() {
     }
 
     /**
-     * Validates the specified {@link Revision} and converts it into an absolute {@link Revision}.
+     * Returns the {@link CompletableFuture} whose value is the absolute {@link Revision} of the
+     * specified {@link Revision}. This will validate the specified {@link Revision} before converting it.
+     *
+     * @see #normalizeNow(Revision)
      */
     CompletableFuture<Revision> normalize(Revision revision);
```

Comment:
Could be a default method like: 

```java
try {
    return CompletableFuture.completedFuture(normalizeNow(revision));
} catch (Exception e) {
    return CompletableFutures.exceptionallyCompletedFuture(e);
}
```

and then we can remove `GitRepository.normalize()`